starter-code/public/javascripts/financial-data.js

```diff
@@ -0,0 +1,68 @@
+axios.get('http://api.coindesk.com/v1/bpi/historical/close.json')
+  .then(response => {
+    printTheChart(response.data.bpi);
+  })
+  .catch(error => {
+    console.log(error);
+  });
+
+
+const printTheChart = (bitCoinData => {
+  const bitCoinLabels = Object.entries(bitCoinData).map(element => element[0]);
+  const bitCoinPrice = Object.entries(bitCoinData).map(element => element[1]);
```

Comment:
You can achieve the same with less code using `Object.keys(bitCoinData)` and `Object.values(bitCoinData)`